temporalcli/commandsmd/commands.md

```diff
@@ -57,6 +57,43 @@ This document has a specific structure used by a parser. Here are the rules:
 * `--color` (string-enum) - Set coloring. Options: always, never, auto. Default: auto.
 * `--no-json-shorthand-payloads` (bool) - Always all payloads as raw payloads even if they are JSON.
 
+### temporal activity: Complete or fail an activity.
+
+#### Options
+
+Includes options set for [client](#options-set-for-client).
+
+
+### temporal activity complete: Complete an activity.
+
+Complete an Activity Execution.
+
+`temporal activity complete --activity-id=MyActivityId --workflow-id=MyWorkflowId --result='{"MyResultKey": "MyResultVal"}'`
+
+#### Options
+
+* `--activity-id` (string) - The Activity to be completed. Required.
+* `--identity` (string) - Identity of operator.
```

Comment:
It was a bit inconsistent for us to have our CLI accept identity here but not on other commands like start workflow. I wonder if we should accept `--identity` higher up.

Reply:
Agreed; I'd like consistency across all protos that have an `identity` field. I was just going for parity here.